src/magento-payments/onboarding-payments.md

```diff
@@ -1,3 +1,129 @@
+Need help accessing your My Account dashboard? Check out our [Create a Magento account topic]({% link magento/magento-account-create.md %}).
+
+A given API key pair is valid for all Magento SaaS services in an environment, so if you already have [Magento Services]({% link system/saas.md %}) configured for your Magento instance your API key pair is already present in the Admin. If your private API key is lost, a new API key pair must be generated and applied to the Magento Services configuration in the Admin.
+
+See the [Magento Services topic]({% link system/saas.md%}) to learn how to generate an API key for either sandbox or production environments.
+
+## Configure Magento Services
+
+The first step in onboarding Magento Payments is to configure your Magento Services in the Admin.
+
+1. On the _Admin_ sidebar go to **Sales** > **Magento Payments**.
+1. Click <span class="btn">Configure Magento Services</span>.
+
+   This option will be visible if you have not yet configured Magento Services for your Magento account.
+
+   You will be directed to the configuration area in the Admin, **Stores** > _Settings_ > **Configuration** > **Magento Services**, to configure your Magento Services.
+
+1. Follow the steps in the [Create Saas Environment]({% link system/saas.md%}#createsaasenv) section of the Magento Services topic to create a SaaS environment (configure Magento Services).
+
+## Enable sandbox testing
+
+There are two options for completing sandbox onboarding:
+
+* **Sign up for a free PayPal Developer's account** and _then_ proceed with our sandbox onboarding flow in the Payments Admin. You can create multiple accounts using this method.
+* **Create a sandbox account on-the-fly** during the onboarding process. This method does not require you to log into PayPal prior, but does require a valid email address.
+
+Either option will aid you in completing onboarding.
+
+To complete sandbox onboarding:
+
+1. On the _Admin_ sidebar go to **Sales** > **Magento Payments**.
+1. Click <span class="btn">Sandbox onboarding</span>.
+
+   This option will be visible if you have not yet completed sandbox onboarding for Magento Payments.
+
+   You are directed to the [PayPal Developer Account page](https://developer.paypal.com/developer/accounts/).
+
+1. Click <span class="btn">Log in to Dashboard</span> and log into an existing PayPal Developers account or click <span class="btn">Sign Up</span> to create a new one.
+1. To create a sandbox account:
+   1. Go to _SANDBOX_ > **Accounts**.
+   1. Click <span class="btn">Create account</span>.
+   1. Select **Business** as the Account Type and click <span class="btn">Create</span>.
+   1. In the _Sandbox Accounts_ section, click the three dots in the _Manage accounts_ column for the sandbox account you just created.
+   1. Click **View/edit account**.
+
+      ![PayPal - View/edit sandbox account]({% link magento-payments/assets/onboarding-vieweditsandbox.png %}){: .zoom}
+      _View/edit sandbox account_
```

Comment:
probably don't want to include the password in the screenshot

Reply:
Yes, you are right. I have a separate story to rejigger all screenshots as needed.